Marlin/Configuration.h

```diff
@@ -3267,6 +3268,8 @@
   //#define RGB_LED_G_PIN 43
   //#define RGB_LED_B_PIN 35
   //#define RGB_LED_W_PIN -1
+  #define RGB_STARTUP_ANIMATION // If pwm pins, fade between all colors if not only switch.
+  #define RGB_STARTUP_ANIM_SLOWING_MS 10 // (ms) Reduce or increase fading speed.
```

Comment:
`RBG_STARTUP_TEST` would be a better name to match the NeoPixel option below of the same name/function. Also, these should be disabled by default of not fully supported on all HALs.

Reply:
Yes , I'm french and my english not very 'interesting' to find the perfect names , it's why , I give the code , and the master here can review all comments/description/names/var names.
I make some corrections for travis checking and , if you have time or want to make a review, feel free to do this

Thks for help